warehouse/templates/manage/organization/projects.html

```diff
@@ -31,6 +31,12 @@ <h2>{% trans project_count=active_projects|length %}Projects ({{ project_count }
       <div class="split-layout split-layout--no-wrap split-layout--wrap-on-tablet">
         <div>
           <h3 class="package-snippet__title">{{ project.name }}
+            {% if project.organizations %}
+            <a href="{{ request.route_path('manage.organization.projects', organization_name=project.organizations[0].normalized_name) }}" class="badge package-snippet__badge">
+              <i class="fa fa-users" aria-hidden="true"></i>
+              {{ project.organizations[0].name }}
```

Comment:
The relationship looks because we are using an `organization_project` association table, but there will only ever be a single organization for each project.

Reply:
To avoid confusion in the future, I’ve changed the `Project.organizations` relationship to `Project.organization` with `uselist=False`.